root/components/SortableTableHeader.js

```diff
@@ -0,0 +1,40 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import React from 'react';
+
+import {withCatalystContext} from '../context';
+import uriWith from '../utility/uriWith';
+
+function printSortArrows(name, order) {
+  if (order === name) {
+    return ' ▴';
+  } else if (order === '-' + name) {
+    return ' ▾';
+  }
+  return ' ▴/▾';
```

Comment:
We currently don't show anything if there's no order set, but that means the fact these are orderable is basically a hidden feature. So this is my attempt at unhiding that.

Reply:
Sounds like a good idea, though it looks a bit busy now. Maybe we should set 50% opacity when it's ▴/▾? That's what a few other table sort plugins I've looked at do.

Reply:
0.5 seemed a bit too strong still, 0.4 seemed about right to me, so did that.